tethys_gizmos/gizmo_options/highcharts_plot_view.py

```diff
@@ -1,1285 +0,0 @@
-# coding=utf-8
```

Comment:
Maybe have placeholders that throw an error and points them to instructions to updating to the new method so they can update it. Do we know how many apps are using the old method?